scripts/install_frontend_tests_dependencies.sh

```diff
@@ -1,30 +0,0 @@
-#!/usr/bin/env bash
-
-# Copyright 2018 The Oppia Authors. All Rights Reserved.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#      http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS-IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-set -e
-source $(dirname $0)/setup.sh || exit 1
-
-install_node_module jasmine-core 2.5.2
```

Comment:
Not included in package.json? (Here and elsewhere.)

Reply:
Added to package.json